request/configure.go

```diff
@@ -11,7 +11,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func configure(request request) {
+func configure(request *request) {
```

Comment:
Why the change to a pointer? Efficiency?

Reply:
yeah, I forgot that by default the structures are copied when being passed over as parameters. I noticed this when writing a test :)